src/main/java/frc/robot/commands/vision/NodeShifter.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.vision;
+import frc.robot.Constants;
+import frc.robot.subsystems.Vision;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class NodeShifter extends CommandBase {
+  /** Creates a new NodeShifter. */
+  private final DrivetrainSubsystem drivetrainSubsystem;
+  private final Vision vision;
+  boolean leftOrRight;
+  public NodeShifter(Vision vision, DrivetrainSubsystem drivetrainSubsystem, boolean leftOrRight) {
```

Comment:
It would be best to name this variable something like shiftRight or shiftLeft as with a name leftOrRight its not clear which direction "true" / "false" mean